src/test/kotlin/uk/gov/justice/digital/hmpps/hmppsintegrationapi/controllers/v1/PersonControllerTest.kt

```diff
@@ -98,10 +96,10 @@ internal class PersonControllerTest(
     it("returns a person with matching first and last name") {
       val result = mockMvc.perform(get("$basePath?first_name=$firstName&last_name=$lastName")).andReturn()
 
-      result.response.contentAsString.shouldBe(
+      result.response.contentAsString.shouldContain(
         """
           {
-            "persons":
+            "content":
```

Comment:
Does this have to change? Can it be kept as `persons`? Also feel like I see `data` more than `content`... 🤔